app/components/Footer.tsx

```diff
@@ -1,7 +1,93 @@
+"use client";
+
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+import { Github, Linkedin } from "lucide-react";
+
+type FooterNavLinkProps = {
+  href: string;
+  text: string;
+};
+
+const FooterNavLink = ({ href, text }: FooterNavLinkProps) => {
+  const currentRoute = usePathname();
+
+  const shouldHaveUnderline =
+    currentRoute.includes(href.slice(1)) ||
+    (currentRoute === "/" && href === "/#about");
+
+  return (
+    <Link
+      href={href}
+      className="text-neutral-500 hover:text-neutral-100 w-fit whitespace-nowrap transition-all duration-300"
+    >
+      <span>{text}</span>
+      <div
+        className={`w-6 h-[1px] ${
+          shouldHaveUnderline ? "bg-neutral-500" : "bg-neutral-900"
+        }`}
+      ></div>
+    </Link>
+  );
+};
+
+const links = [
+  { path: "/#about", text: "About" },
+  { path: "/publications", text: "Publications" },
+  { path: "/contact", text: "Contact Us" },
+  { path: "/join", text: "Join Our Team" },
+];
+
 const Footer = () => {
   return (
-    <footer className="p-4 bg-background text-center text-stone-900">
-      Footer Text
+    <footer className="px-4 pb-6 pt-16 bg-neutral-900 text-center text-neutral-500">
```

Comment:
So, comment on design. Your desktop layout is perfect, love it. For mobile design though we typically rearrange a few things.

![image](https://github.com/Barnstorm-Research/brc-web/assets/13735131/d21f12c3-c3fa-4b96-ae24-ad1876581ff1)

Usually in mobile, Links are king and appear above everything else. Also they are left justified. Then below that is the company logo with social links, centered. And then at the bottom is the copyright stuff, also centered.

So, could you please try establishing a grid layout or some css method for handling that transition from desktop to mobile that keeps links to the right in desktop but places them on top for mobile. CSS grid might not be the best solution here, so definitely please use whatever is easiest.